rendermodules/em_montage_qc/schemas.py

```diff
@@ -79,3 +79,43 @@ class DetectMontageDefectsParametersOutput(argschema.schemas.DefaultSchema):
         dtype='object',
         required=True,
         description='An array of (x,y) positions of seams for each section with seams')
+
+
+
+class RoughQCSchema(RenderParameters):
+    input_downsampled_stack = Str(
+        required=True,
+        description="Pre rough aligned downsampled stack")
+    output_downsampled_stack = Str(
+        required=True,
+        description="Rough aligned stack name")
+    minZ = Int(
+        required=True,
+        description="min z")
+    maxZ = Int(
+        required=True,
+        description="max z")
+    pool_size = Int(
+        required=False,
+        default=10,
+        missing=10,
+        description="Pool size")
+    output_dir = Str(
+        required=False,
+        default=None,
+        missing=None,
+        description="temp filename to save fig")
+    out_file_format = Int(
```

Comment:
would a string argument be more clear here?

Reply:
It doesn't matter... First time users will always have to check the schema help descriptions for accepted values of the input. But changed this for anyway for our convenience.